Auto update failing for weechat

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [weechat](https://github.com/termux/termux-packages/blob/272b8388f31646c1ff7e455600a7a8703e30403b/packages/weechat) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating weechat [Current version: 4.4.3]
INFO: package being updated to 4.4.4.
Downloading https://www.weechat.org/files/src/weechat-4.4.4.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2666k  100 2666k    0     0  2051k      0  0:00:01  0:00:01 --:--:-- 2052k

You are about to commit these changes:

--------------------
diff --git a/packages/weechat/build.sh b/packages/weechat/build.sh
index 3193bae34a..0618592696 100644
--- a/packages/weechat/build.sh
+++ b/packages/weechat/build.sh
@@ -4,9 +4,9 @@ TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
 # `weechat-python-plugin` depends on libpython${TERMUX_PYTHON_VERSION}.so.
 # Please revbump and rebuild when bumping TERMUX_PYTHON_VERSION.
-TERMUX_PKG_VERSION="4.4.3"
+TERMUX_PKG_VERSION="4.4.4"
 TERMUX_PKG_SRCURL=https://www.weechat.org/files/src/weechat-${TERMUX_PKG_VERSION}.tar.xz
-TERMUX_PKG_SHA256=295612f8dc24af28c918257d3014eb53342a5d077d5e3d9a3eadf303bd8febfa
+TERMUX_PKG_SHA256=a8f4bb768c3d6ac3ea1eb4e6dc7a7bb2ee19b734a72cc58e063476dae8f3d077
 TERMUX_PKG_DEPENDS="libandroid-support, libcurl, libgcrypt, libgnutls, libiconv, ncurses, zlib, zstd"
 TERMUX_PKG_BREAKS="weechat-dev"
 TERMUX_PKG_REPLACES="weechat-dev"
--------------------

bump(main/weechat): 4.4.4


Not committing to Git!

INFO: Trying to build package.
INFO: Package weechat exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building weechat for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0   241k      0 --:--:-- --:--:-- --:--:--  242k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2359      0 --:--:-- --:--:-- --:--:--  2359
gpg: Signature made Sat 30 Nov 2024 03:17:46 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  33969      0 --:--:-- --:--:-- --:--:-- 33979
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2385      0 --:--:-- --:--:-- --:--:--  2380
100   833  100   833    0     0   2385      0 --:--:-- --:--:-- --:--:--  2380
gpg: Signature made Thu 28 Nov 2024 12:30:35 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  37318      0 --:--:-- --:--:-- --:--:-- 37378
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2145      0 --:--:-- --:--:-- --:--:--  2146
gpg: Signature made Fri 29 Nov 2024 07:58:05 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libandroid-support@29 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp2@1.64.0 if necessary...
Found libnghttp2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libnghttp2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp3@1.6.0 if necessary...
Found libnghttp3 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libnghttp3 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2024.09.24 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2024.09.24 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.3.2 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.3.2 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libssh2@1.11.1 if necessary...
Found libssh2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libssh2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcurl@8.11.0-1 if necessary...
Found libcurl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcurl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency curl@8.11.0-1 if necessary...
Found curl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting curl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgpg-error@1.50 if necessary...
Found libgpg-error in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libgpg-error/libgpg-error_1.50_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  111k  100  111k    0     0  1312k      0 --:--:-- --:--:-- --:--:-- 1329k
extracting libgpg-error to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgcrypt@1.11.0 if necessary...
Found libgcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libgcrypt/libgcrypt_1.11.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  478k  100  478k    0     0  4557k      0 --:--:-- --:--:-- --:--:-- 4597k
extracting libgcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libc++@27c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgmp@6.3.0-1 if necessary...
Found libgmp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libgmp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.17 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.17 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libunistring@1.3 if necessary...
Found libunistring in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libu/libunistring/libunistring_1.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  535k  100  535k    0     0  3422k      0 --:--:-- --:--:-- --:--:-- 3432k
extracting libunistring to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libidn2@2.3.7 if necessary...
Found libidn2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libi/libidn2/libidn2_2.3.7_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  102k  100  102k    0     0  1681k      0 --:--:-- --:--:-- --:--:-- 1684k
extracting libidn2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnettle@3.10 if necessary...
Found libnettle in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libn/libnettle/libnettle_3.10_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  396k  100  396k    0     0  4595k      0 --:--:-- --:--:-- --:--:-- 4613k
extracting libnettle to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency nettle@3.10 if necessary...
Found nettle in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/nettle/nettle_3.10_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 18152  100 18152    0     0   305k      0 --:--:-- --:--:-- --:--:--  310k
extracting nettle to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libevent@2.1.12-2 if necessary...
Found libevent in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libe/libevent/libevent_2.1.12-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  194k  100  194k    0     0  2060k      0 --:--:-- --:--:-- --:--:-- 2049k
extracting libevent to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libexpat@2.6.4 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.5.20240831-2 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.5.20240831-2 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.5.20240831-2 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.5.20240831-2 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.13 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.24 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.6-1 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.6.3 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.6.3 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.14 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.47.1 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.47.1 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite-tcl@3.47.1 if necessary...
Found libsqlite-tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite-tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.1.0 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.44 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.3 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.15.0 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.15.0 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.2 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2024.1 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.11 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.5 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.17.0-1 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.17.0-1 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.17.0 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.5.2 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.10 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.11 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxft@2.3.8 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.6 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxss@1.2.4 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.14 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.12.7-1 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.12.7-1 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.12.7-1 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency resolv-conf@1.3 if necessary...
Found resolv-conf in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting resolv-conf to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2@10.44 if necessary...
Found pcre2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2grep@10.44 if necessary...
Found pcre2grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2grep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency swig@4.3.0 if necessary...
Found swig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/swig/swig_4.3.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  909k  100  909k    0     0  12.0M      0 --:--:-- --:--:-- --:--:-- 12.1M
extracting swig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libunbound@1.22.0 if necessary...
Found libunbound in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libu/libunbound/libunbound_1.22.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 16  351k   16 60878    0     0  1047k      0 --:--:-- --:--:-- --:--:-- 1043k
100  351k  100  351k    0     0  4834k      0 --:--:-- --:--:-- --:--:-- 4819k
extracting libunbound to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency unbound@1.22.0 if necessary...
Found unbound in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/u/unbound/unbound_1.22.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  578k  100  578k    0     0  5943k      0 --:--:-- --:--:-- --:--:-- 5968k
extracting unbound to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pyunbound@1.22.0 if necessary...
Found pyunbound in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pyunbound/pyunbound_1.22.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 39236  100 39236    0     0   777k      0 --:--:-- --:--:-- --:--:--  781k
extracting pyunbound to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgnutls@3.8.5 if necessary...
Found libgnutls in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libgnutls/libgnutls_3.8.5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  711k  100  711k    0     0  9575k      0 --:--:-- --:--:-- --:--:-- 9608k
extracting libgnutls to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gnutls@3.8.5 if necessary...
Found gnutls in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gnutls/gnutls_3.8.5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  172k  100  172k    0     0  2890k      0 --:--:-- --:--:-- --:--:-- 2915k
extracting gnutls to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblua53@5.3.6 if necessary...
Found liblua53 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libl/liblua53/liblua53_5.3.6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 84444  100 84444    0     0  1043k      0 --:--:-- --:--:-- --:--:-- 1057k
extracting liblua53 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency lua53@5.3.6 if necessary...
Found lua53 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/l/lua53/lua53_5.3.6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 88512  100 88512    0     0  1372k      0 --:--:-- --:--:-- --:--:-- 1372k
extracting lua53 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-utimes@0.4 if necessary...
Found libandroid-utimes in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-utimes/libandroid-utimes_0.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  3120  100  3120    0     0  64573      0 --:--:-- --:--:-- --:--:-- 65000
extracting libandroid-utimes to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency perl@5.38.2-3 if necessary...
Found perl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/perl/perl_5.38.2-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13.5M  100 13.5M    0     0   100M      0 --:--:-- --:--:-- --:--:--  101M
extracting perl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-execinfo@0.1-1 if necessary...
Found libandroid-execinfo in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-execinfo to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libyaml@0.2.5-4 if necessary...
Found libyaml in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liby/libyaml/libyaml_0.2.5-4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 43774  100 43774    0     0   799k      0 --:--:-- --:--:-- --:--:--  806k
extracting libyaml to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ruby@3.3.5 if necessary...
Found ruby in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/r/ruby/ruby_3.3.5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 7203k  100 7203k    0     0  15.9M      0 --:--:-- --:--:-- --:--:-- 15.9M
extracting ruby to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ruby-ri@3.3.5 if necessary...
Found ruby-ri in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/r/ruby-ri/ruby-ri_3.3.5_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  1 1791k    1 30626    0     0  53215      0  0:00:34 --:--:--  0:00:34 53170
100 1791k  100 1791k    0     0  1645k      0  0:00:01  0:00:01 --:--:-- 1646k
extracting ruby-ri to /home/builder/.termux-build/_cache-all...
Downloading dependency zstd@1.5.6-2 if necessary...
Found zstd in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zstd to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pzstd@1.5.6-2 if necessary...
Found pzstd in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pzstd to /home/builder/.termux-build/_cache-aarch64...
Downloading https://www.weechat.org/files/src/weechat-4.4.4.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 49 2666k   49 1327k    0     0  1199k      0  0:00:02  0:00:01  0:00:01 1198k
100 2666k  100 2666k    0     0  2204k      0  0:00:01  0:00:01 --:--:-- 2203k
Applying patch: cmake-FindPerl.cmake.patch
Applying patch: no-setuid.patch
Applying patch: src-core-core-config.c.patch
Applying patch: src-core-core-url.c.patch
Applying patch: src-core-hook-hook-url.c.patch
Applying patch: src-core-hook-hook-url.h.patch
Applying patch: src-core-weechat.c.patch
Applying patch: src-gui-curses-CMakeLists.txt.patch
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The C compiler identification is Clang 18.0.3
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Found PkgConfig: /home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/pkg-config (found version "2.3.0")
-- Looking for include file langinfo.h
-- Looking for include file langinfo.h - found
-- Looking for include file sys/resource.h
-- Looking for include file sys/resource.h - found
-- Looking for include file malloc.h
-- Looking for include file malloc.h - found
-- Looking for malloc_trim
-- Looking for malloc_trim - not found
-- Looking for mallinfo
-- Looking for mallinfo - found
-- Looking for mallinfo2
-- Looking for mallinfo2 - not found
-- Looking for eat_newline_glitch
-- Looking for eat_newline_glitch - found
-- Checking for module 'libgcrypt'
--   Found libgcrypt, version 1.11.0-unknown
-- Found ZLIB: /data/data/com.termux/files/usr/lib/libz.so (found version "1.3.1")
-- Checking for module 'libzstd'
--   Found libzstd, version 1.5.6
-- Looking for libiconv_open in /data/data/com.termux/files/usr/lib/libiconv.so
-- Looking for libiconv_open in /data/data/com.termux/files/usr/lib/libiconv.so - found
-- Looking for iconv_open in /data/data/com.termux/files/usr/lib/libiconv.so
-- Looking for iconv_open in /data/data/com.termux/files/usr/lib/libiconv.so - not found
-- Performing Test ICONV_2ARG_IS_CONST
-- Performing Test ICONV_2ARG_IS_CONST - Failed
-- Found CURL: /data/data/com.termux/files/usr/lib/libcurl.so (found version "8.11.0")
-- Looking for include file libintl.h
-- Looking for include file libintl.h - found
-- Looking for dgettext
-- Looking for dgettext - not found
-- Found Intl: built in to C library
-- Looking for flock
-- Looking for flock - found
-- Looking for backtrace
-- Looking for backtrace - not found
-- Checking for module 'python3-embed'
--   Found python3-embed, version 3.12
CMake Error at src/plugins/CMakeLists.txt:140 (message):
  Ruby not found


-- Could NOT find Ruby (missing: Ruby_INCLUDE_DIR Ruby_CONFIG_INCLUDE_DIR) (found suitable version "3.2.3", minimum required is "1.8.0")
-- Checking for one of the modules 'lua;lua5.4;lua-5.4;lua54;lua5.3;lua-5.3;lua53;lua5.2;lua-5.2;lua52;lua5.1;lua-5.1;lua51;lua-5.0;lua5.0;lua50'
-- Checking for one of the modules 'ncursesw'
-- Looking for include file ncursesw/ncurses.h
-- Looking for include file ncursesw/ncurses.h - found
CMake Warning (dev) at src/gui/curses/normal/CMakeLists.txt:67 (add_custom_command):
  Exactly one of PRE_BUILD, PRE_LINK, or POST_BUILD must be given.  Assuming
  POST_BUILD to preserve backward compatibility.

  Policy CMP0175 is not set: add_custom_command() rejects invalid arguments.
  Run "cmake --help-policy CMP0175" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Configuring incomplete, errors occurred!
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-11-30 12:17:41 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/12096230520">12096230520</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/weechat/build.sh
+++ b/packages/weechat/build.sh
@@ -4,9 +4,9 @@ TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
 # `weechat-python-plugin` depends on libpython${TERMUX_PYTHON_VERSION}.so.
 # Please revbump and rebuild when bumping TERMUX_PYTHON_VERSION.
-TERMUX_PKG_VERSION="4.4.3"
+TERMUX_PKG_VERSION="4.5.1"
 TERMUX_PKG_SRCURL=https://www.weechat.org/files/src/weechat-${TERMUX_PKG_VERSION}.tar.xz
-TERMUX_PKG_SHA256=295612f8dc24af28c918257d3014eb53342a5d077d5e3d9a3eadf303bd8febfa
+TERMUX_PKG_SHA256=67c143c7bc70e689b9ea86df674c9a9ff3cf44ccc9cdff21be6a561d5eafc528
 TERMUX_PKG_DEPENDS="libandroid-support, libcurl, libgcrypt, libgnutls, libiconv, ncurses, zlib, zstd"
 TERMUX_PKG_BREAKS="weechat-dev"
 TERMUX_PKG_REPLACES="weechat-dev"
@@ -38,4 +38,17 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 
 termux_step_pre_configure() {
 	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" -DPKG_CONFIG_EXECUTABLE=${PKG_CONFIG}"
+
+	local _Ruby_API_VERSION=$(
+		. $TERMUX_SCRIPTDIR/packages/ruby/build.sh
+		echo "$(echo $TERMUX_PKG_VERSION | cut -d . -f 1-2).0"
+	)
+	local _Ruby_INCLUDE_DIR="$TERMUX_PREFIX/include/ruby-$_Ruby_API_VERSION"
+	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" -DRuby_INCLUDE_DIR=$_Ruby_INCLUDE_DIR"
+	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" -DRuby_CONFIG_INCLUDE_DIR=$_Ruby_INCLUDE_DIR"
+	if [ "$TERMUX_ARCH" == "arm" ]; then
+		TERMUX_PKG_EXTRA_CONFIGURE_ARGS+="/$TERMUX_ARCH-linux-androideabi"
+	else
+		TERMUX_PKG_EXTRA_CONFIGURE_ARGS+="/$TERMUX_ARCH-linux-android"
+	fi
 }

